src/content/blog/2024-08-22-understanding-the-ugrc-api-for-address-locating.mdx

```diff
@@ -0,0 +1,47 @@
+import { Image } from 'astro:assets';
+
+The [UGRC API](https://api.mapserv.utah.gov/) is a valuable resource for [locating an address](https://api.mapserv.utah.gov/docs/v1/endpoints/geocoding/). We are proud to offer this service to you for free, however, it's not the only solution for your address locating needs. Let's explore the layers that enable the `/geocode` endpoint to help you understand what this service offers and its limitations.
+
+## The foundational data
+
+The UGRC provides authoritative GIS data for Utah through the [SGID](/products/sgid/sgid-index/). By aggregating data from local governments and standardizing it, we are able to offer this comprehensive statewide resource. Our address locating endpoint provides excellent address match rates because of the collaborative efforts of our local address authority partners and their dedicated work. Together, we maintain two foundational SGID datasets that power the UGRC API -- [address points](/products/sgid/location/address-points/) and [road centerlines](/products/sgid/transportation/road-centerlines/). [Zip codes](/products/sgid/boundaries/zip-codes/) and [address systems](/products/sgid/location/address-system-quadrants/) participate in the API and are worth mentioning but we won't be discussing them further.
+
+The road centerlines and address point data is regularly updated, with the "Big 5" counties (Salt Lake, Utah, Davis, Weber, Washington) providing monthly updates. While the other counties provide updates on a less frequent schedule, the "Big 5" cover approximately 80% of Utah's population, ensuring great coverage in densely populated areas. Stay informed about the latest updates by following the [SGID Update](/blog/categories/sgid-updates/) blog category.
+
+## Road centerlines versus address points
+
+Road centerlines and address points are two different datasets that work together to provide the best address location match rates. By default, both datasets are used to provide the solution and address points have priority in the case of a tie. The UGRC API provides which dataset solved your request in the response and you are able to specify which data participates in the match using the `locator` optional input. [Read the documentation](https://api.mapserv.utah.gov/docs/v1/endpoints/geocoding/) for a more complete look at the options available to tune the results. This is important since the road centerlines are used to determine the location of an address along a road segment, while the address points are used to determine the exact location of an address. By using both datasets together, you will get the best match rate the data can provide. But a match on the road centerlines has a different meaning than a match on the address points. But if you are zooming, routing, extracting data, or looking for a generalized location for analysis a match on the road centerlines is a great option.
```

Comment:
```suggestion
Road centerlines and address points are two different datasets that work together to provide the best address location match rates. By default, both datasets are used to provide the solution and address points have priority in the case of a tie. The UGRC API provides which dataset solved your request in the response and you are able to specify which data participates in the match using the `locator` optional input. [Read the documentation](https://api.mapserv.utah.gov/docs/v1/endpoints/geocoding/) for a more complete look at the options available to tune the results. This is important since the road centerlines are used to determine the location of an address along a road segment, while the address points are used to determine the exact location of an address. 

By using both datasets together, you will get the best match rate the data can provide. But a match on the road centerlines has a different meaning than a match on the address points. But if you are zooming, routing, extracting data, or looking for a generalized location for analysis a match on the road centerlines is a great option.
```

Reply:
not critical, just thought it might break up the long paragraph and make it easier to digest.